$OpenBSD: patch-Source_JavaScriptCore_assembler_MacroAssemblerX86Common_h,v 1.2 2013/10/06 14:03:42 landry Exp $
./Source/JavaScriptCore/assembler/MacroAssembler.h:909: error: call of overloaded 'shouldBlindForSpecificArch(uintptr_t&)' is ambiguous
./Source/JavaScriptCore/assembler/MacroAssemblerX86Common.h:98: note: candidates are:
     static bool JSC::MacroAssemblerX86Common::shouldBlindForSpecificArch(uint32_t)
./Source/JavaScriptCore/assembler/MacroAssemblerX86Common.h:100: note: 
     static bool JSC::MacroAssemblerX86Common::shouldBlindForSpecificArch(uint64_t)

--- Source/JavaScriptCore/assembler/MacroAssemblerX86Common.h.orig	Fri Aug 23 08:26:41 2013
+++ Source/JavaScriptCore/assembler/MacroAssemblerX86Common.h	Wed Aug 28 21:16:22 2013
@@ -102,7 +102,7 @@ class MacroAssemblerX86Common : public AbstractMacroAs
     static bool shouldBlindForSpecificArch(uint32_t value) { return value >= 0x00ffffff; }
 #if CPU(X86_64)
     static bool shouldBlindForSpecificArch(uint64_t value) { return value >= 0x00ffffff; }
-#if OS(DARWIN) // On 64-bit systems other than DARWIN uint64_t and uintptr_t are the same type so overload is prohibited.
+#if OS(DARWIN) || OS(OPENBSD) // On 64-bit systems other than DARWIN uint64_t and uintptr_t are the same type so overload is prohibited.
     static bool shouldBlindForSpecificArch(uintptr_t value) { return value >= 0x00ffffff; }
 #endif
 #endif
